internal/controllers/provisioningrequest_controller_test.go

```diff
@@ -1052,10 +1052,10 @@ var _ = Describe("ProvisioningRequestReconcile", func() {
 			// Create Non-compliant enforce policy
 			policy = &policiesv1.Policy{
 				ObjectMeta: metav1.ObjectMeta{
-					Name:      "ztp-clustertemplate-a-v4-15.v1-sriov-configuration-policy",
+					Name:      "ztp-clustertemplate-a-v4-16.v1-sriov-configuration-policy",
```

Comment:
In the spirit of our examples, maybe these should be actually renamed to `v4.Y`

Reply:
An exact match would be `v4.Y.Z`.  It might be good to have but is not a must-have. It would be a seperate PR to update all testcases if needed.